packages/sanity/src/structure/panes/documentList/sheetList/__tests__/DocumentSheetListPane.test.tsx

```diff
@@ -298,15 +413,15 @@ describe('DocumentSheetListPane', () => {
     it('should render column headers', async () => {
```

Comment:
COULD: add another test here for the change in functionality to the header ellipsis menu - now we support the 'Remove from table' button on parents of column groups. We could test that removing the parent group, eg translations, removes all the group columns - eg norwegian, english